.stylelintrc.json

```diff
@@ -0,0 +1,21 @@
+{
+    "extends": [
```

Comment:
Be careful not to add too much indentation to CSS, it always looks better having less separation margin and is good for semantics and good code practice, the rest was well done, 
🏆 congrats!!🏆

Reply:
This looks impressive :+1: 

_There's still room for improvement:_ 

- I'll implore you to check out how to master [markdowns for writing comments](https://guides.github.com/features/mastering-markdown/) on GitHub. This is because I can see that you're trying to use markdown syntax, but it's not done in the proper way. Taking some minutes to master this will allow you to write better and expressive comments subsequently :muscle: 

- Also, you've suggested that the student should not add too much indentation, which is a great catch :blush:. But it will be better if you can specify the exact number of space required for indentation, or better still, you can refer the student to a genuine writeup/article/stack-overflow page to support your claim.

Reply:
thank you :)